ramani-maplibre/src/main/java/org/ramani/compose/LocationStyling.kt

```diff
@@ -61,12 +87,13 @@ class LocationStyling(
         if (accuracyColor != other.accuracyColor) return false
         if (enablePulse != other.enablePulse) return false
         if (enablePulseFade != other.enablePulseFade) return false
-        if (pulseColor == other.pulseColor) return false
-        if (bearingTintColor == other.bearingTintColor) return false
-        if (foregroundTintColor == other.foregroundTintColor) return false
-        if (backgroundTintColor == other.backgroundTintColor) return false
-        if (foregroundStaleTintColor == other.foregroundStaleTintColor) return false
-        if (backgroundStaleTintColor == other.backgroundStaleTintColor) return false
+        if (pulseColor != other.pulseColor) return false
+        if (bearingTintColor != other.bearingTintColor) return false
+        if (foregroundTintColor != other.foregroundTintColor) return false
+        if (backgroundTintColor != other.backgroundTintColor) return false
+        if (foregroundStaleTintColor != other.foregroundStaleTintColor) return false
+        if (backgroundStaleTintColor != other.backgroundStaleTintColor) return false
```

Comment:
This causes issues with the LaunchedEffect, making it run in a loop and eventually crashing the app.